source/pymongo.txt

```diff
@@ -99,6 +99,19 @@ If the connection fails, you should see the following message:
 For more information on the connection string, see the MongoDB Server
 Manual entry on :manual:`Connection String URI Format </reference/connection-string/>`.
 
+Versioned API
+-------------
+
+You can use the `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`__ 
+feature starting with MongoDB Server version 5.0 and Pymongo driver
```

Comment:
I think we agreed to capitalize Driver in this sentence, to keep consistency with the capitalization of Server.

Reply:
From reading how it's used throughout the site, it doesn't look like we have a consistent way to refer to it. I do think "MongoDB Server" is a product, but haven't decided if "PyMongo Driver" is the product name or if it's just PyMongo, the Python driver.
Let's keep the capitalization of driver here (in this case, a change), but not make any changes across the rest of the docs.